Exiled.API/Features/Door.cs

```diff
@@ -264,14 +274,19 @@ public Vector3 Scale
         /// </summary>
         public ZoneType Zone => Room?.Zone ?? ZoneType.Unspecified;
 
+        /// <summary>
+        /// Gets a <see cref="List{T}"/> containing all <see cref="Features.Room"/>'s that are connected with <see cref="Door"/>.
+        /// </summary>
+        internal List<Room> RoomsValue { get; } = new List<Room>();
+
         /// <summary>
         /// Gets the door object associated with a specific <see cref="DoorVariant"/>, or creates a new one if there isn't one.
         /// </summary>
         /// <param name="doorVariant">The base-game <see cref="DoorVariant"/>.</param>
         /// <returns>A <see cref="Door"/> wrapper object.</returns>
         public static Door Get(DoorVariant doorVariant) => doorVariant != null ? (DoorVariantToDoor.TryGetValue(doorVariant, out Door door)
             ? door
-            : new Door(doorVariant, doorVariant.GetComponentInParent<Room>())) : null;
+            : new Door(doorVariant, null)) : null;
```

Comment:
can we make something like that ?
```new List<Room>() {doorVariant.GetComponentInParent<Room>(),}```

Reply:
In general you right, but i do it like that because of complex reason:
during checkpoint door init in `GetDoorType` method, children checkpoint door was inited and added to a list too, so it led to problems. 
with `null` param door wont be added to list, so yeah